nixos/doc/manual/installation/installing-usb.section.md

```diff
@@ -1,31 +1,49 @@
-# Booting from a USB Drive {#sec-booting-from-usb}
+# Booting from a USB flash drive {#sec-booting-from-usb}
 
-For systems without CD drive, the NixOS live CD can be booted from a USB
-stick. You can use the `dd` utility to write the image:
-`dd if=path-to-image of=/dev/sdX`. Be careful about specifying the correct
-drive; you can use the `lsblk` command to get a list of block devices.
+The image has to be written verbatim to the USB flash drive for it to be
+bootable on UEFI and BIOS systems. Here are the recommended tools to do that.
 
-::: {.note}
-::: {.title}
-On macOS
+## Creating bootable USB flash drive with a graphical tool {#sec-booting-from-usb-graphical}
+
+Etcher is a popular and user-friendly tool. It works on Linux, Windows and macOS.
+
+Download it from [balena.io](https://www.balena.io/etcher/), start the program,
+select the downloaded NixOS ISO, then select the USB flash drive and flash it.
+
+::: {.warning}
+Etcher reports errors and usage statistics by default, which can be disabled in
+the settings. Some users might not like that.
 :::
 
+An alternative is [USBImager](https://gitlab.com/bztsrc/usbimager/#usbimager),
+which is very simple and does not connect to the internet. Download the version
+with write-only (wo) interface for your system. Start the program,
+select the image, select the USB flash drive and click "Write".
+
+## Creating bootable USB flash drive from Terminal on Linux {#sec-booting-from-usb-linux}
+
+Use the `dd` utility to write the image to the USB flash drive:
+
 ```ShellSession
-$ diskutil list
-[..]
-/dev/diskN (external, physical):
-   #:                       TYPE NAME                    SIZE       IDENTIFIER
-[..]
-$ diskutil unmountDisk diskN
-Unmount of all volumes on diskN was successful
-$ sudo dd if=nix.iso of=/dev/rdiskN bs=1M
+sudo dd if=<path-to-image> of=/dev/sdX bs=4M conv=fsync oflag=direct status=progress
```

Comment:
We might not want to add the `bs`, `conv`, `oflag` and `status` parameters as they might not be present in all `dd` flavours used by distros.

Also, other commenters: check the manpage for `dd` and cite from it if you have any concerns about the params. A lot of "common knowledge" around `dd` is cargo-culted and not exactly right.

Reply:
I actually just copied the command from Arch Wiki. The progress information seem useful to me, but i can remove that again for compatibility.